web/src/components/tasks/business-formation/review/ReviewMainBusinessLocation.tsx

```diff
@@ -11,33 +10,57 @@ export const ReviewMainBusinessLocation = (): ReactElement => {
-    return getStringifiedAddress({
-      addressLine1: state.formationFormData.addressLine1 || italicNotEnteredText,
-      city:
-        (businessLocationType === "NJ"
-          ? state.formationFormData.addressMunicipality?.displayName
-          : state.formationFormData.addressCity) || italicNotEnteredText,
-      state:
-        (businessLocationType === "INTL"
-          ? state.formationFormData.addressProvince
-          : state.formationFormData.addressState?.name) || italicNotEnteredText,
-      zipcode: state.formationFormData.addressZipCode || italicNotEnteredText,
-      addressLine2: state.formationFormData.addressLine2,
-      country:
-        businessLocationType === "INTL"
-          ? arrayOfCountriesObjects.find(
-              (country) => country.shortCode === state.formationFormData.addressCountry
-            )?.name ?? italicNotEnteredText
-          : undefined,
-    });
+  const getAddressCity = (): string => {
+    return (
+      (businessLocationType === "NJ"
+        ? state.formationFormData.addressMunicipality?.displayName
+        : state.formationFormData.addressCity) || italicNotEnteredText
+    );
+  };
+
+  const getAddressState = (): string => {
+    return (
+      (businessLocationType === "INTL"
+        ? state.formationFormData.addressProvince
+        : state.formationFormData.addressState?.name) || italicNotEnteredText
+    );
+  };
+
+  const getAddressCountry = (): string | undefined => {
+    return businessLocationType === "INTL"
+      ? arrayOfCountriesObjects.find(
+          (country) => country.shortCode === state.formationFormData.addressCountry
+        )?.name ?? italicNotEnteredText
+      : undefined;
   };
 
   return (
     <ReviewSubSection header={Config.formation.sections.review.locationHeader} marginOverride="margin-top-0">
-      <ReviewLineItem label={Config.formation.sections.review.addressLabel} value={getAddressDisplay()} />
+      <ReviewLineItem
+        label={Config.formation.fields.addressLine1.label}
+        value={state.formationFormData.addressLine1 || italicNotEnteredText}
+      />
+      {state.formationFormData.addressLine2 && (
+        <ReviewLineItem
+          label={Config.formation.fields.addressLine2.label}
+          value={state.formationFormData.addressLine2}
+          dataTestId={"business-address-line-two"}
+        />
+      )}
+      <ReviewLineItem label={Config.formation.fields.addressCity.label} value={getAddressCity()} />
+      <ReviewLineItem label={Config.formation.fields.addressState.label} value={getAddressState()} />
+      <ReviewLineItem
+        label={Config.formation.fields.addressZipCode.label}
+        value={state.formationFormData.addressZipCode || italicNotEnteredText}
+      />
+      {businessLocationType === "INTL" && (
+        <ReviewLineItem
+          label={Config.formation.fields.addressState.label}
+          value={getAddressCountry() || ""}
```

Comment:
Instead of handling an undefined return type here, could the getter be responsible for that?